scripts/build/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "build",
+  "version": "0.5.7",
+  "private": true,
+  "dependencies": {
+    "chalk": "^2.3.2",
+    "del": "^3.0.0",
+    "rollup": "^0.62.0",
+    "rollup-plugin-alias": "^1.4.0",
+    "rollup-plugin-babel": "^4.0.0-beta.5",
```

Comment:
it's safer to pin beta packages to exact versions, also - this could get upgraded, there is `beta.7` already